model_types/modelTypesList.json

```diff
@@ -48,18 +51,6 @@
         "bytes": 100000000
       }
     },
-    {
-      "modelType": "evaluate",
-      "defaultConfig": "config_templates/gretel/evaluate/default.yml",
-      "sampleDataset": {
-        "fileName": "bank_marketing_small.csv",
-        "description": "Create synthetic data based on the publicly available dataset predicting opting in or out of bank marketing.",
-        "records": 4521,
-        "fields": 17,
-        "trainingTime": "< 10 mins",
-        "bytes": 371020
-      }
-    },
     {
       "modelType": "transform",
       "defaultConfig": "config_templates/gretel/transform/default.yml",
```

Comment:
**suggestion:** would you mind adding the category to transform and classify, too, so we can stay consistent with the data shape for now?